vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -127,13 +127,17 @@ testPackagesInstalled() {
 
       # if the downloadLocation is /usr/local/bin verify that the package is installed
       if [ "$downloadLocation" == "/usr/local/bin" ]; then
-          if command -v "$name" >/dev/null 2>&1; then
-              echo "$name is installed."
-              continue
-          else
-              err $test "$name is not installed. Expected to be installed in $downloadLocation"
-              continue
-          fi
+        if command -v "$name" >/dev/null 2>&1; then
+          echo "$name is installed."
+          continue
+        elif [[ "$name" == containerd-wasm-* ]]; then
+          testWasmRuntimesInstalled
```

Comment:
add a condition for `"spinkube"`

Reply:
the name in components.json is containerd-wasm-spinkube so it should cover spinkube